modules/ossm-about-concepts-resources.adoc

```diff
@@ -78,6 +76,32 @@ You can think of the relationship between the `Istio` and `IstioRevision` resour
 
 Similarly, users create an `Istio` resource which instructs the {SMProduct} Operator to create a matching `IstioRevision` resource, which then in turn triggers the creation of the Istio control plane. To do that, the {SMProduct} Operator will copy all of your relevant configuration from the `Istio` resource to the `IstioRevision` resource.
 
+[id="istiorevisiontag-resource_{context}"]
+== IstioRevisionTag resource
+
+The `IstioRevisionTag` resource represents a stable revision tag, which functions as an alias for Istio control plane revisions. With a stable tag `prod`, you can use the label `istio.io/rev=prod` to inject proxies into your workloads. When you perform an upgrade to a control plane with a new revision name, you can update your tag to point to the new revision instead of having to re-label your workloads and namespaces. For more information, see "Stable Revision Tags."
+
+In Istio, stable revision tags are usually created using `istioctl` client. If you are using the {SMProduct} Operator, you can use the `IstioRevisionTag` resource, which lets you reference both an `IstioRevision` and an `Istio` resource. Now, when you update your control plane and the underlying `IstioRevision` changes, the {SMProduct} Operator updates your revision tag for you. You only need to restart your deployments to re-inject the new proxies.
```

Comment:
"Now, when you update your control plane..." this applies only when referencing `Istio` resource, it should be described clearly here.